packages/core/src/lib/store.ts

```diff
@@ -19,10 +19,21 @@ const reducer = (
     case "SETUP_WALLET_MODULES": {
       const { modules, accounts, contract, selectedWalletId } = action.payload;
 
+      const activeAccountIndex = state.accounts.findIndex(
```

Comment:
We don't need this logic for `SETUP_WALLET_MODULES` as we won't have an active account. We can always assume the first account, unless we decided to persist the active account in the future